localization/english/wc_laws_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+ dark_authority:0 "[dark_authority|E]"
+ dark_authority_0:0 "Disparate Cults"
+ dark_authority_1:0 "Weak Covens"
```

Maybe replace weak with something that corresponds with influence more.
```suggestion
 dark_authority_1:0 "Insignificant Circles"
```